.github/workflows/helm-docs.yaml

```diff
@@ -23,7 +23,6 @@ jobs:
           passphrase: ${{ secrets.GPG_COMMITS_PASSPHRASE }}
           git-user-signingkey: true
           git-commit-gpgsign: true
-          commit_options: '--signoff'
```

Comment:
Looks accidental

Reply:
Nope, is not. I fixed the signing because the option was placed at the wrong place/ action.